pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -62,6 +62,7 @@ type ReconcileChaosEngine struct {
 	// recorder is an event recorder for recording Event resources to the
 	// Kubernetes API.
 	recorder record.EventRecorder
+	ctx context.Context
```

Comment:
Where is this `ctx` variable set.? While creating the struct you can set it as may be `context.TODO()`  or `context.Background()`